src/components/apps/launch/params/FolderInput.js

```diff
@@ -9,14 +9,23 @@ import InputSelector from "../InputSelector";
 
 import ResourceTypes from "components/models/ResourceTypes";
 
+import { useBootstrapInfo } from "contexts/bootstrap";
+
 export default function FileFolderInput({ param, ...props }) {
+    const bootstrapInfo = useBootstrapInfo()[0];
+    const pref = bootstrapInfo?.preferences;
     return (
         <InputSelector
             margin="normal"
             acceptedType={ResourceTypes.FOLDER}
             label={param?.label}
             helperText={param?.description}
             required={param?.required}
+            startingPath={
+                pref?.rememberLastPath
+                    ? pref?.lastFolder
+                    : pref?.defaultFileSelectorPath
+            }
```

Comment:
Also not needed here, since the `InputSelector` should already be doing this internally when `startingPath` is not set.